text/0088-fix-memory-limitiations-in-session-replays-access-pattern.md

```diff
@@ -0,0 +1,288 @@
+- Start Date: 2023-04-24
+- RFC Type: feature
+- RFC PR: https://github.com/getsentry/rfcs/pull/88
+- RFC Status: draft
+
+# Summary
+
+The Session Replay index page will run out of memory when processing queries for our largest customers. This document contains proposed solutions for addressing this short-coming.
+
+# Motivation
```

Comment:
What is the expected timeline of this? Those constraints would be good to know when evaluating solutions